optoforce/launch/optoforce_hand.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="rviz" default="true"/>
+
+  <include file="$(find optoforce)/launch/optoforce.launch"/>
+
+  <include if="$(arg rviz)" file="$(find optoforce)/launch/rviz.launch"/>
+
+  <!-- Static transforms form Optoforce sensor to finger tip -->
+  <node name="optoforce_1_broadcaster" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 1.57 rh_rftip optoforce_1 100" />
```

Comment:
Well, we can perhaps move this launchfile to sr_config.
We don't know what sensor will be mounted on what finger.
Also this driver for the moment only covers the 4-sensor USB module, so there isn't one sensor per finger.